pkg/admin/types.go

```diff
@@ -73,6 +75,221 @@ type PartitionAssignment struct {
 	Replicas []int `json:"replicas"`
 }
 
+// PartitionInfo represents the information stored about an ACL
+// in zookeeper.
+type ACLInfo struct {
+	ResourceType   ResourceType
+	ResourceName   string
+	PatternType    PatternType
+	Principal      string
+	Host           string
+	Operation      ACLOperationType
+	PermissionType ACLPermissionType
+}
+
+// ResourceType presents the Kafka resource type.
+// We need to subtype this to be able to define methods to
+// satisfy the Value interface from Cobra so we can use it
+// as a Cobra flag.
+type ResourceType kafka.ResourceType
```

Comment:
These types allow us to convert between kafka-go types (most of which are int8s that represent something in the Kafka API) and human read able forms of the types for the user to provide through the CLI or see displayed as output